sunxspex/emission.py

```diff
@@ -558,15 +820,99 @@ def brm2_dmlino(a, b, maxfcn, rerr, eph, eelow, eebrk, eehigh, p, q, z):
     return DmlinO, ier
 
 
-def bremstralung_thicktarget(eph, p, eebrk, q, eelow, eehigh):
+def Brm2_ThinTarget(eph, p, eebrk, q, eelow, eehigh):
```

Comment:
Trying to make the names more descriptive perhaps `bremsstrahlung_thin_target` also python function usually use snake case.

Reply:
done.